src/renderers/react/ElementWrapper.tsx

```diff
@@ -9,18 +9,28 @@ import { TelemetryContext } from "./TelemetryContext";
 import { LeftActionControls, RightActionControls } from "./WrapperControls";
 
 export const Container = styled("div")`
-  margin: ${space[3]}px -${actionSpacing}px;
+  margin: ${space[3]}px 0;
+  position: relative;
 `;
 
 export const Body = styled("div")`
   display: flex;
-  :hover,
-  :focus-within {
-    .actions {
-      opacity: 1;
-    }
-  }
   min-height: 134px;
+  .nested & {
```

Comment:
I think the extra vertical space is not too problematic, but we can maybe shave a few pixels by adding some negative margin here.